Dockerfile

```diff
@@ -45,22 +118,16 @@ RUN mv node_modules ./migrate/node_modules
 # Copy temp node_modules of app to app folder
 RUN mv _node_modules node_modules
 
-RUN echo '#!/bin/bash\nnode /app/cli/cli.js "$@"' > /usr/bin/homarr
-RUN chmod +x /usr/bin/homarr
 RUN cd /app/cli && yarn --immutable
 
-# Expose the default application port
-EXPOSE $PORT
-ENV PORT=${PORT}
+# Root is needed for supervisord
+USER root
 
-ENV DATABASE_URL "file:/data/db.sqlite"
-ENV NEXTAUTH_URL "http://localhost:7575"
-ENV PORT 7575
-ENV NEXTAUTH_SECRET NOT_IN_USE_BECAUSE_JWTS_ARE_UNUSED
+RUN echo '#!/bin/bash\nnode /app/cli/cli.js "$@"' > /usr/bin/homarr
+RUN chmod +x /usr/bin/homarr
```

Comment:
Better to use an `ENTRYPOINT ["/path/to/node", "/app/cli/cli.js"]` instead, the `"$@"` is unnecessary as you get this functionality from `ENTRYPOINT` with the syntax shown, it will append `CMD` to it, or whatever a user provides as an override to `CMD` at runtime.

If you use the distroless image suggestion, you don't have to worry about the `ENTRYPOINT`, it'll point to node already, but [they suggest to put the args into `CMD`](https://github.com/GoogleContainerTools/distroless/blob/dad24cbf02268737b7cbd8ee7ae4cfb973e84203/examples/nodejs/Dockerfile#L10), which may be a slight breaking change to the image if someone was providing custom args to `cli.js`.